docs/cow-protocol/tutorials/solvers/flashloans_support.md

```
@@ -0,0 +1,87 @@
+
+Together with the support for flashloans we want to minimize the order filtering the autopilot does before even building the auction. This was implemented as a performance optimization since most of the orders do not have sufficient balance. The order filtering is already implemented in the reference driver so nothing should change for solvers connected to that. For colocated drivers the auction in the `/solve` endpoint will contain ~3K orders or more going forward.
+
+## Details on AAVE’s Integration
+
+AAVE will cover 3 new use cases:
+
+1. Collateral swap
+
+User has a debt position and wants to change the collateral they use.
+
+Example: https://explorer.cow.fi/gc/orders/0x413a7246f58441ad92ea19c09cef90d1b23a1e211e0963f3d39b7db48140533d669685c660c260d80b614f8d1a5ffd24c4e3b82668cd8760
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)